components/toponyme/toponyme-numeros.js

```diff
@@ -10,6 +10,12 @@ function ToponymeNumeros({numeros, handleSelect, isEditable}) {
     return groupBy(numeros.sort((a, b) => a.numero - b.numero), d => d.voie.nom)
   }, [numeros])
 
+  const handleClick = id => {
+    if (isEditable) {
+      handleSelect(id)
+    }
+  }
```

Comment:
Cette méthode étant utilisée dans la liste des numéros, il est préférable d'utiliser une `useCallback` afin de limiter de rendus.

```suggestion
  const handleClick = useCallback(id => {
    if (isEditable) {
      handleSelect(id)
    }
  }, [isEditable, handleSelect])
```